src/Mod/Part/App/Geometry.cpp

```diff
@@ -793,6 +793,11 @@ GeomBSplineCurve* GeomCurve::toNurbs(double first, double last) const
     return toBSpline(first, last);
 }
 
+GeomCurve* GeomCurve::createArc([[maybe_unused]] double first, [[maybe_unused]] double last) const
+{
+    return nullptr;

```

Comment:
If this is intentional please leave a comment why it only returns nullptr. Maybe it should be abstract function instead? 

This is not a function that I'd expect to return nullptr in any scenario - if it is expected to fail it should use `std::optional` if not - it should be abstract (`=0`).

Reply:
I'm hesitant to use `=0` since there is not other abstract method in GeomCurve.

Reply:
I'm not sure how this class is used, but returning `nullptr` here could cause bugs if someone does not know that such method must be implemented. If not `=0` maybe exception? Throwing exception seems better than causing segfault.